spec/resources/aws_iam_saml_provider_spec.rb

```diff
@@ -10,7 +10,9 @@
     before do
       list_resp = double()
       saml_resp = double()
-      allow(saml_resp).to receive(:arn).and_return("arn::some:stuf:here:lol")
+      allow(saml_resp).to receive(:arn).and_return(
```

Comment:
nit you should use the AWS mocks as they validate the returned values

Reply:
Have en example of a mock?

Reply:
oh i am using a stub, below on line 17